sonarr/type.go

```diff
@@ -9,6 +9,29 @@ import (
 // APIver is the Sonarr API version supported by this library.
 const APIver = "v3"
 
+// DelayProfile is the /api/v3/delayprofile endpoint.
```

Comment:
Later I'm going to break up the type.go files, and move the structs to their most-appropriate file. Feel free to do that with this contribution; put these types in the new files you made instead of here.

Reply:
great! just moved!